### DISCUSSION TITLE
Chart add a "Rest"

### DISCUSSION THREAD

**Discussion by thewebartisan7** [orchidsoftware/platform#2067] [Q&A] [answered]
I have created a chart and I can see an added value "Rest" which is not coming from my data.

Any idea what is this?

<img width="654" alt="Screenshot 2022-01-16 at 07 52 24" src="https://user-images.githubusercontent.com/57477934/149650430-247f48b8-b496-4231-9d40-bab948d11df3.png">

My array passed as target is like below:

<img width="224" alt="Screenshot 2022-01-16 at 07 57 11" src="https://user-images.githubusercontent.com/57477934/149650489-3f65c49e-d7d8-43da-9a4a-40c1bc788def.png">

Also I have 10 different labels/values but there is only 7 shown in pie chart, and one of this "Rest" it's not mine.

**Comment by tabuna** [comment, marked as answer]
Charts have a division limit which is 7 by default. You can change it in your class https://github.com/orchidsoftware/platform/blob/beb8e1eb258af95ea1c8238614203573a1e73319/src/Screen/Layouts/Chart.php#L84


If there is more data than the set amount, then some will be combined into a group called "rest". https://github.com/frappe/charts/blob/d075c76ee2c259f6ecb3126e7397597c79b3bc01/src/js/charts/AggregationChart.js#L43

**Reply by thewebartisan7** [reply]
@tabuna thanks it worked!

There is a way to set the height of SVG because in small screen the label go outside the box, see screenshot attached.

I could play with CSS, but maybe there is a way via orchid?

<img width="1089" alt="Screenshot 2022-01-17 at 12 31 36" src="https://user-images.githubusercontent.com/57477934/149762467-aa680f46-8120-4f81-8505-dd0a15d57d3e.png">
